src/services/ldap/strategies/general.js

```diff
@@ -142,7 +143,12 @@ class GeneralLDAPStrategy extends AbstractLDAPStrategy {
 	getClasses(school) {
 		const { classAttributeNameMapping, classPathAdditions } = this.config.providerOptions;
 
-		if (classPathAdditions !== '') {
+		if (
+			isNotEmptyString(classPathAdditions, true) &&
```

Comment:
the issue mentioned in the ticket was to handle `"undefined"` (the string with undefined as it's value) not `undefined` _type_ from javascript.

Reply:
that happens because the search string will contain the undefined as a string
`${classPathAdditions},${this.config.rootPath}`; 
and is handled by fixing the condition above

Reply:
what is the value of classPathAdditions here?

Reply:
undefined, and in concatenation searchString becomes "undefined,${this.config.rootPath}"...

Reply:
`classPathAdditions` this necessary to set the place on the ldap tree where to search the ldap objects for classes.